Framework/include/QualityControl/DataSourceSpec.h

```diff
@@ -39,9 +39,9 @@ enum class DataSourceType {
 // this should allow us to represent all data sources which come from DPL (and maybe CCDB).
 struct DataSourceSpec {
   explicit DataSourceSpec(DataSourceType type = DataSourceType::Invalid);
-  // todo: use c++20 concepts when available
-  template <class... Args, class Enable = std::enable_if_t<(... && std::is_convertible_v<Args, DataSourceType>)>>
-  bool isOneOf(Args... dataSourceType) const
+
+  template <class... DataSourceType>
```

Comment:
shouldn't there be any concept similarly to `computeNumericalActivityField` ?

Reply:
Why? this method is a member of `DataSourceSpec` and expects `DataSourceType` only, but it can accept one or more of those.

Do you mean just saying something like `requires T == DataSourceType`)?

Reply:
the "todo" comment was probably my lack of awareness, that it could be done simpler.

Reply:
well, there was a structure that checks whether the template args can be converted to the enum, so I thought that you want to keep the functionality

Reply:
no, not really. i think i had thought that one needs `is_convertible` to make it work also with `const DataSourceType` or `const DataSourceType&`, which apparently is not needed in the end.